run.sh

```diff
@@ -1,22 +1,33 @@
+#!/bin/bash
+
 # note: must be run from the project root folder
 
+# set env variables
+envFile="$PWD/.env"
+source "$envFile"
+export BOOTSTRAP_SERVER=${DOCKER_HOST_IP}:9092
```

Comment:
Our update on docker-compose.yml changes this to BOOTSTRAP_SERVER, but here we're trying to read in the DOCKER_HOST_IP still. Should this be tweaked?

Reply:
Updated this to pull from the BOOTSTRAP_SERVER env variable + updated sample.env to include BOOTSTRAP_SERVER